docs/source/reference/spawners.md

```diff
@@ -119,7 +119,7 @@ def clear_state(self):
 
 Some deployments may want to offer options to users to influence how their servers are started.
 This may include cluster-based deployments, where users specify what resources should be available,
-or docker-based deployments where users can select from a list of base images.
+or docker-based deployments where admins can select from a list of base images.
```

Comment:
The original is correct, users select from a list of base images (the list is configured by the admin)

Reply:
ok thanks will revert